python/interpret_community/explanation/explanation.py

```diff
@@ -460,12 +460,15 @@ def get_raw_explanation(self, feature_maps, raw_feature_names=None):
             in a single element list can be passed
         :param raw_feature_names: list of raw feature names
         :type raw_feature_names: [str]
+        :param eval_data: Evaluation data.
+        :type eval_data: pd.DataFrame
         :return: raw explanation
         :rtype: LocalExplanation
         """
         raw_kwargs = _get_raw_explainer_create_explanation_kwargs(explanation=self)
         raw_kwargs[ExplainParams.FEATURES] = raw_feature_names
         raw_kwargs[ExplainParams.IS_RAW] = True
+        raw_kwargs[ExplainParams.EVAL_DATA] = eval_data
```

Comment:
maybe add some tests for the new param

Reply:
Added tests that check if the eval_data has been correctly set on the explanation object